src/test/java/org/apache/pulsar/ecosystem/io/lakehouse/source/delta/SourceContextForTest.java

```diff
@@ -146,4 +147,8 @@ public CompletableFuture<Long> getCounterAsync(String key) {
     public void recordMetric(String metricName, double value) {
         metrics.put(metricName, value);
     }
+
+    public SourceConfig getSourceConfig() {
```

Comment:
Do we need to mark `@Override` ?

Reply:
We don't need the override because the method does not exist in the previous version. And even if the `@Override` doesn't exist, the code is work well.

Reply:
@hangc0276